.circleci/config.yml

```diff
@@ -13,7 +13,11 @@ cache_keys: &cache_keys
     # Rev the version when the cache gets too big
     - trace-java-v1-{{ .Branch }}-{{ .Revision }}
     - trace-java-v1-{{ .Branch }}
-    # - dd-trace-java-v1-
+
+parameters:
+  gradle_flags:
+    type: string
+    default: "--build-cache --parallel --stacktrace --no-daemon"
```

Comment:
I think it is better idea to put such common properties into project source code: https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties